html2text/__init__.py

```diff
@@ -420,7 +421,11 @@ def no_preceding_space(self: HTML2Text) -> bool:
                 self.stressed = True
 
         if tag in ["strong", "b"] and not self.ignore_emphasis:
-            if start and no_preceding_space(self):
+            if (
```

Comment:
Would be great to have some reasoning documented here (_documenting WHY_) so reading code would be easier.

Reply:
Good call! Thanks, done.